package.json

```diff
@@ -18,6 +18,8 @@
     "babel-register": "6.14.0",
     "eslint": "3.5.0",
     "eslint-config-kentcdodds": "10.1.0",
+    "eslint-plugin-babel": "^3.3.0",
+    "eslint-plugin-mocha": "^4.5.1",
```

Comment:
Actually, I must have missed this when I did my big refactor, but we don't need the `mocha` plugin anymore, could you replace `"kentcdodds/mocha"` with `"kentcdodds/jest"` on [this line](https://github.com/kentcdodds/es6-workshop/blob/e1a683ac2c6e030e2fafb3c269dadcd25b8ab317/package.json#L46)?

Then you can remove this dep. Keep the babel plugin. Thanks a bunch!